### DISCUSSION TITLE
Tree view for issues

### DISCUSSION THREAD

**Discussion by viraptor** [ankitpokhrel/jira-cli#774] [Ideas]
I'd love to see the issues as a tree where they are connected.
In a plan view in JIRA I can see something like
```
Epic foo
+- Story bar
|  +- task
|     +- subtask
|  +- other task
+- Story baz
...
```

The issue list already knows about the type of the items, but when looking at a specific project, it would be great to see the dependencies like above.

**Comment by jamesbebbington** [comment]
I came here looking for this feature too. Being able to pass in an arbitrary issue key and then see the issue hierarchy from its epic down would be really nice.
